packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -259,7 +259,11 @@ class Vif extends BaseComponent {
       props.pool === undefined // to get objects as a self user
     ),
     srs: getSrs(state, props),
-    template: getTemplate(state, props),
+    template: getTemplate(
+      state,
+      props,
+      props.location.query.resourceSet !== undefined
```

Comment:
For homogeneity with the rest of the code: test `props.pool === undefined` instead.